GeoDaCenter/chicago-environment-explorer
# feat: explicitly choose category order

## Problem
We would like to be able to explicitly re-order the list of categories

Currently, they appear in alphabetical order

Fixes #138 

## Approach
* feat: added a new Categories sheet that allows us to explicitly re-order the category list

![Screenshot 2024-04-04 at 11 21 25 AM](https://github.com/GeoDaCenter/chicago-environment-explorer/assets/1413653/b830240e-2da3-43d3-99a0-b4fcb96ac3b0)

![Screenshot 2024-04-04 at 11 27 58 AM](https://github.com/GeoDaCenter/chicago-environment-explorer/assets/1413653/8c978487-021a-4d81-834c-5239bb8138d8)


## How to Test
1. Navigate to /builder
    * You should see that Air Quality appears first with Heat Indicators just below
1. Navigate to /map and expand the Variable dropdown on the VariablePanel (left side)
    * You should see that Air Quality appears first with Heat Indicators just below
